src/components/Common/AuthorizedButton.tsx

```diff
@@ -1,9 +1,11 @@
-import { Button, ButtonProps } from "@headlessui/react";
+import * as React from "react";
 
 import AuthorizedChild from "@/CAREUI/misc/AuthorizedChild";
 
 import { AuthorizedElementProps } from "@/Utils/AuthorizeFor";
 
+import { Button, ButtonProps } from "../ui/button";
```

Comment:
minor thing, but switch to absolute import. Same thing for button import in other files.